### PR TITLE
Create Happiness Engineers Tray Component

### PR DESCRIPTION
**Pull request by Aurorum**
#### Changes proposed in this Pull Request

This is the first step to #38998. With that, the row of Happiness Engineers' Gravatars would be used three times throughout Calypso, with each use case requiring the same implementation and styling. It probably makes sense to just create a component for it. 

#### Testing instructions

There's two cases to test. Firstly, check `/hosting-config/site`, but remove the `feature-example__gradient` class if you don't have a Business Plan.

Screenshot 2020-02-29 at 20 17 11

Secondly, check the bottom of `/help`
Screenshot 2020-02-29 at 20 17 18

The third case will be with #38998.

cc @mmtr @gwwar

### CODE REVIEW

**Comment by Aurorum** [comment]
I'm unsure why the tests are failing and how to fix them

**Comment by mmtr** [comment]
> I'm unsure why the tests are failing and how to fix them

We have a known issue in the `icfy-stats` build, so feel free to ignore that. The failures on `test-client` seems irrelevant as well, although the error message doesn't provide any helpful info.

If you want to be sure, rather than adding a random commit or merging with master, could you try rebasing onto master and see if that makes the CI to pass?

```
git remote add upstream git@github.com:Automattic/wp-calypso.git
git fetch upstream
git rebase upstream/master
git push origin happiness-engineers-tray --force
```

File: client/components/happiness-engineers-tray/style.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.happiness-engineers-tray {
+	display: flex;
+	flex-wrap: wrap;
+	height: 42px;
+	justify-content: space-between;
+	margin-bottom: 16px;
```
Comment (mmtr) [inline client/components/happiness-engineers-tray/style.scss]: Adding a negative margin right here should remove the extra right space we have now.
```suggestion
	margin-bottom: 16px;
	margin-right: -8px;
```

File: client/my-sites/hosting/support-card/style.scss [hunk 2]
```diff
@@ -1,21 +0,0 @@
...
-		margin-bottom: 16px;
-		margin-right: -8px;
-
-		.gravatar {
-			margin-right: 8px;
-		}
-	}
-
-	p {
-		margin-bottom: 16px;
```
Comment (mmtr) [inline client/my-sites/hosting/support-card/style.scss]: We should maintain this style rule for paragraphs, so all elements in the support card are separated by 16px.

**Comment by Aurorum** [comment]
Thanks @mmtr! I've fixed both styling issues which you mentioned. I'm having a bit less luck with the rebasing; even after following the steps you suggested, I'm still running into issues. I'm trying to see if I can sort it out. :)

**Comment by mmtr** [comment]
> I'm having a bit less luck with the rebasing; even after following the steps you suggested, I'm still running into issues

No worries! Maybe tests will pass now after your not-so-random commit 🙂

**Comment by Aurorum** [comment]
Sadly the tests are still failing!

**Comment by mmtr** [comment]
> Sadly the tests are still failing!

Looks like I have write access to your branch, so I went ahead and rebased the branch for you. Let's see if CI is happy now 🤞

**Comment by Aurorum** [comment]
Thanks @mmtr! :)

**Comment by mmtr** [comment]
Same error `Error downloading previous test results`. Let me double check what that CI build is trying to do, but I think this is safe to merge.

**Pull request merged**